enterprise/api/v1/views/enterprise_subsidy_fulfillment.py

```diff
@@ -202,25 +215,19 @@ def unenrolled(self, request, *args, **kwargs):
             retrieve_licensed_enrollments (bool): If true, return data related to licensed enrollments instead of
                 learner credit
         """
-        queryset = self.get_unenrolled_fulfillment_queryset()
+        queryset = self._get_unenrolled_fulfillment_queryset()
         serializer_class = self.get_unenrolled_fulfillment_serializer_class()
         serializer = serializer_class(queryset, many=True)
         return Response(serializer.data)
 
-    @permission_required(
-        'enterprise.can_access_admin_dashboard',
-        fn=lambda request, fulfillment_source_uuid: get_enterprise_customer_from_user_id(request.user.id)
```

Comment:
This was another part of authorization jankiness, it technically didn't "break" anything because of the old queryset filtering above, but what this permission check actually does is ask "does the user have this permission for the enterprise customer they are linked to", but it doesn't take into account _at all_ the customer associated with _the resource being requested_ (a Fulfillment Source record).

Reply:
I feel like typically pylint would flag this as an unused function parameter...